pysqldb3/tests/test_io.py

```diff
@@ -614,32 +637,39 @@ def test_pg_to_pg_qry_basic_table(self):
                                 password=config.get('SECOND_PG_DB', 'DB_PASSWORD'))
 
         # Assert pg table doesn't exist
-        db.drop_table(schema=db.default_schema, table=test_pg_to_pg_qry_table)
-        assert not db.table_exists(table=test_pg_to_pg_qry_table)
+        db.drop_table(schema='working', table=test_pg_to_pg_qry_table)
```

Comment:
please add as variables to top of test file and use as parameters 
`pg_schema = 'working'`
`sql_schema = 'dbo'`